### PR TITLE
CRN-1473  update tutorial model

### PR DESCRIPTION
**Pull request by AkosuaA**
This PR updates the Tutorials model. ref: https://asaphub.atlassian.net/jira/software/c/projects/CRN/boards/6?modal=detail&selectedIssue=CRN-1473.

Includes:
-  Migration script to update Contentful model
-  Updates Tutorials model, queries, data provider, UI components and related tests

### CODE REVIEW

File: apps/crn-server/src/data-providers/contentful/tutorial.data-provider.ts [hunk 1]
```diff
@@ -32,17 +55,109 @@ export class TutorialContentfulDataProvider implements TutorialDataProvider {
...
+        : 'Network Only',
+    tags:
+      tutorial.tagsCollection?.items
+        .map((tag) => tag?.name || '')
+        .filter(Boolean) || [],
+    authors,
+    relatedEvents,
+    teams,
+    relatedTutorials: [
+      ...mapRelatedTutorials(
```
Comment (Riccardo-Brown) [inline apps/crn-server/src/data-providers/contentful/tutorial.data-provider.ts:153]: Can use cleanArray here

File: packages/react-components/src/organisms/TutorialCard.tsx [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+  shortText,
+  created,
+  addedDate,
+  authors,
+  teams,
+  tags,
+}) => {
+  const href = discover({}).tutorials({}).tutorial({ tutorialId: id }).$;
+
+  const newsImage = (
```
Comment (kmaid) [inline packages/react-components/src/organisms/TutorialCard.tsx:99]: tutorial

File: packages/contentful/src/crn/queries/tutorials.queries.ts [hunk 3]
```diff
@@ -43,6 +49,74 @@ export const tutorialsContentQueryFragment = gql`
         }
       }
     }
+    authorsCollection(limit: 100) {
```
Comment (Riccardo-Brown) [inline packages/contentful/src/crn/queries/tutorials.queries.ts:52]: 100 seems high, we probably want to reduce it so the query complexity is not too high.

Comment (AkosuaA) [reply packages/contentful/src/crn/queries/tutorials.queries.ts:52]: I could set the limit at 10 and increase if needed.

**Review by kmaid — APPROVED** [review]
LGTM for FE

File: packages/react-components/src/templates/TutorialDetailsPage.tsx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    props.asapFunded,
+    props.sharingStatus,
+    props.usedInPublication,
+  ].some((info) => info != null);
+
+  return (
+    <div css={containerStyles}>
+      <div css={cardsStyles}>
+        <TutorialHeaderCard {...props} />
+        {(text || !!tags.length) && (
```
Comment (Riccardo-Brown) [inline packages/react-components/src/templates/TutorialDetailsPage.tsx:55]: Why are description and tags dependent on each other? Should it not be we show description if it exist regardless if tags exists and vice versa.

Comment (AkosuaA) [reply packages/react-components/src/templates/TutorialDetailsPage.tsx:55]: We don't want to display the card at all if neither is present

**Pull request merged**